modules/documents/client/views/document-manager.html

```diff
@@ -165,8 +165,11 @@
                 <div class="row-actions">
                   <div class="btn-group">
 
-                    <button class="btn icon-btn dropdown-toggle" type="button" ng-click="$event.originalEvent.dropdown = true" popover-trigger="'focus'"
-                      popover-class="context-menu" uib-popover-template="'file-context-menu.html'" popover-placement="'auto'">
+                    <button class="btn icon-btn dropdown-toggle" type="button" 
```

Comment:
@jeznorth It seems you just reformatted the `<button>` into multiple lines but the new code is missing `ng-click="$event.originalEvent.dropdown = true"` from the original code... Is that intended?

Reply:
That ng-click event was for the previous dropdown menu implementation which is no longer required.

Reply:
👍